components/cards/StateButton.tsx

```diff
@@ -1,5 +1,5 @@
 import Box from '@mui/material/Box'
-import Typography from '@mui/material/Typography/Typography'
+import Typography from '@mui/material/Typography'
```

Comment:
is it called button because it's supposed to be clickable in the future? Or it's more like a badge?

Reply:
it's not clickable, so badge would be a better word for it actually. i'll change that for clarity!